db/seeds.rb

```diff
@@ -6,19 +6,22 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-plan = Plan.create(plan_name: "Daily Success Plan")
+plan = Plan.create(plan_name: 'Daily Success Plan')
 
-habit1 = Habit.create(name: "Rise at 7:00 AM")
-habit2 = Habit.create(name: "Make your bed")
-habit3 = Habit.create(name: "Motivational Mantra")
-habit4 = Habit.create(name: "Update to do list")
+habit1 = Habit.create(name: 'Rise at 7:00 AM')
+habit2 = Habit.create(name: 'Make your bed')
+habit3 = Habit.create(name: 'Motivational Mantra')
+habit4 = Habit.create(name: 'Update to do list')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.